lib/plane.rb

```diff
@@ -0,0 +1,26 @@
+require_relative './airport'
+
+class Plane
+
+  def initialize
+    @current_airport = nil
+    @flying = true
+  end
+
+  def land(airport)
```

Comment:
```suggestion
  def land_at(airport)
```
Using `land_at(airport)` would make this read like a full sentence